chat-plugins/trivia.js

```diff
@@ -686,13 +686,13 @@ var commands = {
 
 			if (isAccepting) {
 				var accepted = [];
-				for (var i = indicesLen; i--;) {
+				for (var i = 0; i < indicesLen; i++) {
```

Comment:
``indices`` is sorted from greatest to least at this point, so it would be splicing at the right offset.